src/styles/mixins/_flex.scss

```diff
@@ -0,0 +1,5 @@
+@mixin page {
+  display: flex;
+  height: 100vh;
```

Comment:
I propose adding some sort of flex-direction: column; here - as things are now the entire layout is in a row (form heading is next to the form, success message displays next to the form instead of under)